pkgs/development/libraries/libappindicator/default.nix

```diff
@@ -1,32 +1,41 @@
 # TODO: Resolve the issues with the Mono bindings.
 
 { stdenv, fetchurl, fetchpatch, lib
-, pkgconfig, autoreconfHook
+, pkg-config, autoreconfHook, automake111x
 , glib, dbus-glib, gtkVersion ? "3"
-, gtk2 ? null, libindicator-gtk2 ? null, libdbusmenu-gtk2 ? null
-, gtk3 ? null, libindicator-gtk3 ? null, libdbusmenu-gtk3 ? null
+, gtk2 ? null, libindicator-gtk2 ? null, libdbusmenu-gtk2 ? null, gnome2
+, gtk3 ? null, libindicator-gtk3 ? null, libdbusmenu-gtk3 ? null, gnome3
 , vala, gobject-introspection
 , monoSupport ? false, mono ? null, gtk-sharp-2_0 ? null
  }:
 
 with lib;
 
-
 stdenv.mkDerivation rec {
   name = let postfix = if gtkVersion == "2" && monoSupport then "sharp" else "gtk${gtkVersion}";
           in "libappindicator-${postfix}-${version}";
-  version = "${versionMajor}.${versionMinor}";
-  versionMajor = "12.10";
-  versionMinor = "0";
+  # pulled from tag, matched formatting with repology
+  version = "12.10.1.201020200706.1";
 
   outputs = [ "out" "dev" ];
 
   src = fetchurl {
-    url = "${meta.homepage}/${versionMajor}/${version}/+download/libappindicator-${version}.tar.gz";
-    sha256 = "17xlqd60v0zllrxp8bgq3k5a1jkj0svkqn8rzllcyjh8k0gpr46m";
+    name = "libappindicator-${version}.tar.gz";
+    url = "https://bazaar.launchpad.net/~indicator-applet-developers/libappindicator/trunk/tarball/298";
+    sha256 = "0l9x2dcmsgp43j4rv5brx4vv2vpz34by8lbl3wp97sircniqkbl6";
   };
 
-  nativeBuildInputs = [ pkgconfig autoreconfHook vala gobject-introspection ];
+  sourceRoot = "~indicator-applet-developers/libappindicator/trunk";
+
+  nativeBuildInputs = [
+    pkg-config
+    autoreconfHook
+    vala
+    gobject-introspection
+    gnome2.gtkdoc # couldn't find a gnome3.gtkdoc related tool
```

Comment:
This is actually an alias for toplevel `gtk-doc`. not sure if it's just the gtk 3 version that needs this?

Reply:
both versions need it, but it's essentially just needed to call `gtkdocize` as part of the `autogen.sh` pre-configure

Reply:
I replaced it with the top-level attr

Reply:
without it, you will get:
```
checking for gtk-doc >= 1.0...
  testing gtkdocize... not found.
***Error***: You must have gtk-doc >= 1.0 installed
  to build libappindicator.  Download the appropriate package for
  from your distribution or get the source tarball at
    http://ftp.gnome.org/pub/GNOME/sources/gtk-doc/
```